### PR TITLE
[FR] Adjustment of certain terms

### PR DESCRIPTION
**Pull request by DoryanR**
Rectification of some terms throughout the wiki for the French articles, in order to have the same terms in all the articles, thus avoiding several terms that all mean the same thing.

The changes concerned by the PR :

`chanson` > `musique`
`chansons` > `musiques`
`map` > `beatmap`
`maps` > `beatmaps`
`mapset` > `beatmap`
`mapsets` > `beatmaps`
`piste` > `something`
`pistes` > `something`
`carte` > `beatmap`
`cartes` > `beatmaps`

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/Menu/fr.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Notez que ces commandes peuvent être désactivées dans les [Options](/wiki/Cli
...
-| Activer le live mapping (`Ctrl` + Tab) | Maps osu!/osu!catch/osu!taiko : utilisez les touches osu!taiko pour saisir des cercles/fruit/taiko, tout en jouant le morceau dans l'éditeur. Les notes sont placées selon l'emplacement du curseur. Différents sons en fonction de la touche osu!taiko sur laquelle vous avez appuyée. maps osu!mania : utilisez les touches osu!mania pour placer des cercles, tout en jouant un morceau dans l'éditeur. La note est placée selon la touche sur laquelle vous avez appuyée. Ne fonctionne pas pour les sliders/holds. |
+| Activer le live mapping (`Ctrl` + Tab) | Beatmaps osu!/osu!catch/osu!taiko : utilisez les touches osu!taiko pour saisir des cercles/fruit/taiko, tout en jouant le morceau dans l'éditeur. Les notes sont placées selon l'emplacement du curseur. Différents sons en fonction de la touche osu!taiko sur laquelle vous avez appuyée. beatmaps osu!mania : utilisez les touches osu!mania pour placer des cercles, tout en jouant un morceau dans l'éditeur. La note est placée selon la touche sur laquelle vous avez appuyée. Ne fonctionne pas pour les sliders/holds. |
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Menu/fr.md:170]:
```suggestion
| Activer le live mapping (`Ctrl` + Tab) | Beatmaps osu!/osu!catch/osu!taiko : utilisez les touches osu!taiko pour saisir des cercles/fruit/taiko, tout en jouant le morceau dans l'éditeur. Les notes sont placées selon l'emplacement du curseur. Différents sons en fonction de la touche osu!taiko sur laquelle vous avez appuyée. Beatmaps osu!mania : utilisez les touches osu!mania pour placer des cercles, tout en jouant un morceau dans l'éditeur. La note est placée selon la touche sur laquelle vous avez appuyée. Ne fonctionne pas pour les sliders/holds. |
```

**Review by DoryanR — COMMENTED** [review]
Self check

**Comment by Biokax** [comment]
not sure about `map > beatmap` and `maps > beatmaps` because some english files contains `map` and `maps`, that seem unnecessary

**Comment by DoryanR** [comment]
Personally I don't understand the use of the term `map`, for me I don't see the difference between `beatmap` and `map`, and since often when you make reviews, you suggest me to change these terms, so I might as well unify the terms.

**Comment by Walavouchey** [comment]
*map* and *beatmap* are the same thing, and they contain *difficulties*.

**Comment by Biokax** [comment]
> _map_ and _beatmap_ are the same thing, and they contain _difficulties_.

If its the same thing it doesn't matter if the translation doesn't follow the exact english terms and replace all maps by beatmaps ?

**Comment by Walavouchey** [comment]
no it doesn't matter

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 2]
```diff
@@ -4,21 +4,21 @@ Song Setup est la quatrième section de [l'éditeur de beatmap](/wiki/Client/Bea
...
 ## Général
 
-### Métadonnées des chansons et des maps
+### Métadonnées des musiques et des beatmaps
 
 *Voir également : [\[Guide\] Metadata](https://osu.ppy.sh/community/forums/topics/249288 "forums osu!")*
 
 ![Un formulaire de Song Setup rempli](img/SS_General.jpg "Un exemple de métadonnées complètes (onglet Général) dans Song Setup")
 
 Ces champs doivent être remplis avant de créer une deuxième difficulté. Il s'agit principalement de s'assurer que les métadonnées seront cohérentes pour toutes les difficultés du beatmapset.
```
Comment (Ganondorf-1) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:13]:
```suggestion
Ces champs doivent être remplis avant de créer une deuxième difficulté. Il s'agit principalement de s'assurer que les métadonnées seront cohérentes pour toutes les difficultés de la beatmap.
```

File: wiki/Guides/Beginner's_Tutorial/fr.md [hunk 3]
```diff
@@ -98,21 +98,21 @@ Ajustez votre global offset jusqu'à ce que vous en soyez satisfait. L'obtention
...
-## Maps pour débutants
+## Beatmaps pour débutants
 
-Essayez d'essayer ces maps. Elles sont classées de la plus facile à la plus difficile, alors essayez de descendre la liste à votre propre rythme !
+Essayez d'essayer ces beatmaps. Elles sont classées de la plus facile à la plus difficile, alors essayez de descendre la liste à votre propre rythme !
 
 ### Débutant
 
-Vous pouvez commencer votre aventure avec ces cartes :
+Vous pouvez commencer votre aventure avec ces beatmap :
```
Comment (Ganondorf-1) [inline wiki/Guides/Beginner's_Tutorial/fr.md:115]:
```suggestion
Vous pouvez commencer votre aventure avec ces beatmaps :
```

File: wiki/Client/Beatmap_editor/Distance_snap/fr.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ no_native_review: true
...
+La façon la plus simple de s'assurer que l'espacement des battements est correct sur vos beatmap est d'utiliser la fonction Distance Snapping de l'éditeur, qui se trouve en haut à droite de l'éditeur. La distance relative entre les notes peut être ajustée en utilisant 'Alt' et la molette de la souris. Cela permet de s'assurer que les temps qui sont proches sur la timeline le sont aussi sur le beatmap, et vice versa. Cela permet un jeu intuitif et un bon espacement des temps. Si vous vous en tenez à l'utilisation de Distance Snap pour tout le morceau, l'espacement correct des temps est garanti !
```
Comment (Ganondorf-1) [inline wiki/Client/Beatmap_editor/Distance_snap/fr.md:11]:
```suggestion
La façon la plus simple de s'assurer que l'espacement des battements est correct sur vos beatmaps est d'utiliser la fonction Distance Snapping de l'éditeur, qui se trouve en haut à droite de l'éditeur. La distance relative entre les notes peut être ajustée en utilisant 'Alt' et la molette de la souris. Cela permet de s'assurer que les temps qui sont proches sur la timeline le sont aussi sur le beatmap, et vice versa. Cela permet un jeu intuitif et un bon espacement des temps. Si vous vous en tenez à l'utilisation de Distance Snap pour tout le morceau, l'espacement correct des temps est garanti !
```

File: wiki/Game_modifier/Auto/fr.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ Dans le mode [osu!](/wiki/Game_mode/osu!), lorsqu'il est confronté à un spinne
 
 ### osu!taiko
 
-Dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko), le mod Auto tentera de jouer parfaitement, mais n'obtiendra pas toujours la note SS. Il manquera parfois un tick sur les drum rolls qui ne commencent pas sur un temps entier ou un demi-ton et ne sera pas toujours capable de terminer les dendens sur les cartes dont la difficulté globale est très élevée.
+Dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko), le mod Auto tentera de jouer parfaitement, mais n'obtiendra pas toujours la note SS. Il manquera parfois un tick sur les drum rolls qui ne commencent pas sur un temps entier ou un demi-ton et ne sera pas toujours capable de terminer les dendens sur les beatmap dont la difficulté globale est très élevée.
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/Auto/fr.md:45]:
```suggestion
Dans le mode [osu!taiko](/wiki/Game_mode/osu!taiko), le mod Auto tentera de jouer parfaitement, mais n'obtiendra pas toujours la note SS. Il manquera parfois un tick sur les drum rolls qui ne commencent pas sur un temps entier ou un demi-ton et ne sera pas toujours capable de terminer les dendens sur les beatmaps dont la difficulté globale est très élevée.
```

File: wiki/Client/Beatmap_editor/AiMod/fr.md [hunk 6]
```diff
@@ -157,19 +157,19 @@ Le développement d'AiMod est généralement en retard par rapport aux normes de
...
+| Ce mapset a besoin d'au moins 2 difficultés osu!taiko. | beatmaps osu!taiko uniquement |  |
 | Les timing points non hérités entrent en conflit avec la difficulté {0}. |  |  |
 
 ## Défauts
 
 <!-- TODO: I'm sure there are more flaws, just can't think of any more off the top of my head -->
 
 - **AiMod ne peut pas détecter un grand nombre de problèmes** qui sont considérés comme non classables par les [critères de classement](/wiki/Ranking_Criteria), tels que les objets potentiellement mal rythmés, les hitsounds retardés, les fichiers d'arrière-plan de plus de 2,5 Mo, etc.
-- **AiMod peut vous avertir de l'absence de problèmes**, comme le fait que les mapsets nécessite deux difficultés du mode osu! sur une map [marathon](/wiki/Beatmap/Marathon) et que l'option [kiai time](/wiki/Gameplay/Kiai_time) soit activée pendant moins de 15 secondes.
+- **AiMod peut vous avertir de l'absence de problèmes**, comme le fait que les beatmaps nécessite deux difficultés du mode osu! sur une beatmap [marathon](/wiki/Beatmap/Marathon) et que l'option [kiai time](/wiki/Gameplay/Kiai_time) soit activée pendant moins de 15 secondes.
```
Comment (Ganondorf-1) [inline wiki/Client/Beatmap_editor/AiMod/fr.md:172]:
```suggestion
- **AiMod peut vous avertir de l'absence de problèmes**, comme le fait que les beatmaps nécessitent deux difficultés du mode osu! sur une beatmap [marathon](/wiki/Beatmap/Marathon) et que l'option [kiai time](/wiki/Gameplay/Kiai_time) soit activée pendant moins de 15 secondes.
```

File: wiki/Guides/Timing_Songs_With_8-Signatures/fr.md [hunk 7]
```diff
@@ -1,22 +1,22 @@
-# Timing des chansons avec des signatures en #/8
+# Timing des musiques avec des signatures en #/8
 
 De temps en temps, des musiques sont mappées avec une signature temporelle qui n'est pas divisée en quart de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en huitièmes de note (par exemple `6/8`, `7/8`, ...). L'éditeur actuel ne prend pas en charge ces signatures #/8 dans un futur proche.
 
 Afin de rendre ce tableau plus compréhensible et d'indiquer plus précisément ce que l'on entend par là, ce guide va maintenant expliquer un peu de théorie musicale pertinente derrière tout cela, et donner des exemples de la façon dont cela se traduit dans osu!. Pour cette raison, [trois beatmapsets](https://assets.ppy.sh/media/wiki/TimeSignatures.rar) ont été préparés pour que vous puissiez les télécharger, et ils seront utilisés comme référence dans ce guide.
```
Comment (Ganondorf-1) [inline wiki/Guides/Timing_Songs_With_8-Signatures/fr.md:5]:
```suggestion
Afin de rendre ce tableau plus compréhensible et d'indiquer plus précisément ce que l'on entend par là, ce guide va maintenant expliquer un peu de théorie musicale pertinente derrière tout cela, et donner des exemples de la façon dont cela se traduit dans osu!. Pour cette raison, [trois beatmaps](https://assets.ppy.sh/media/wiki/TimeSignatures.rar) ont été préparées pour que vous puissiez les télécharger, et ils seront utilisés comme référence dans ce guide.
```

File: wiki/Guides/Timing_Songs_With_8-Signatures/fr.md [hunk 8]
```diff
@@ -26,18 +26,18 @@ En parlant de la difficulté `3/4` de *America* de Leonard Bernstein, j'ai menti
 
 ![150 BPM avec un métronome 2/4 en comparaison avec 100 BPM avec un métronome 3/4](/wiki/shared/timing/150bpm24vs100bpm34.png)
 
-Pour un exemple plus simple, regardez le beatmapset fourni de la chanson *We Are The Champions* de Queen, qui est une chanson purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la chanson, et que tous les huitièmes de note ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.
+Pour un exemple plus simple, regardez le beatmapset fourni de la musique *We Are The Champions* de Queen, qui est une musique purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la musique, et que tous les huitièmes de note ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.
```
Comment (Ganondorf-1) [inline wiki/Guides/Timing_Songs_With_8-Signatures/fr.md:29]:
```suggestion
Pour un exemple plus simple, regardez la beatmap fournie de la musique *We Are The Champions* de Queen, qui est une musique purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la musique, et que tous les huitièmes de note ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.
```

File: wiki/Client/Beatmap_editor/AiMod/fr.md [hunk 9]
```diff
@@ -118,7 +118,7 @@ Le développement d'AiMod est généralement en retard par rapport aux normes de
 | Plus de la moitié de la carte TV Size est constituée d'un kiai time. Envisagez de le réduire. |  |  |
```
Comment (Ganondorf-1) [inline wiki/Client/Beatmap_editor/AiMod/fr.md:118]:
```suggestion
| Plus de la moitié de la beatmap TV Size est constituée d'un kiai time. Envisagez de le réduire. |  |  |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
Is this good to go @Ganondorf-1, @Biokax? I've applied the review

**Comment by Ganondorf-1** [comment]
I can't find any other problem so I think it is

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**